modules/ROOT/pages/cloud/auth/auth.adoc

```diff
@@ -1,6 +1,6 @@
 :page-layout: api-partial
 
-. If <<try-the-cloud-api,issuing requests to the Cloud API from this page>> using the *Try* option, *Get Token*. Do not edit the input field next to the button.
+. If <<try-the-cloud-api,issuing requests to the Cloud API from this page>> using the *Try* option, *Get Token*.
```

Comment:
This wording is confusing to me. Could it be either:
To [issue requests to the Cloud API from this page](https://docs.redpanda.com/api/cloud-api/#try-the-cloud-api), click **Get Token**.
Or
When [issuing requests to the Cloud API from this page](https://docs.redpanda.com/api/cloud-api/#try-the-cloud-api) using the **Try** option, click **Get Token**.